inst/include/interface/rcpp/rcpp_objects/rcpp_interface_base.hpp

```diff
@@ -193,7 +193,7 @@ class ParameterVector{
      *  @param pos return a Parameter at position "pos".
      */
     SEXP at(R_xlen_t pos){
-        if (pos == 0 || pos > this->storage_m->size()) {
+        if (static_cast<size_t>(pos) == 0 || static_cast<size_t>(pos) > this->storage_m->size()) {
```

Comment:
This change should be in #662 not here because it doesn't have anything to do with renaming. I am making a note in the issue now.